UserAdminBundle/Tests/Functional/Controller/ApiControllersSecurityTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace OpenOrchestra\UserAdminBundle\Tests\Functional\Controller;
+
+use OpenOrchestra\ApiBundle\Tests\Functional\Controller\AbstractControllerTest;
+
+/**
+ * Class ApiControllersSecurityTest
+ */
+class ApiControllersSecurityTest extends AbstractControllerTest
+{
+    protected $username = "userLog";
```

Comment:
when this PR https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1152 is merge, use this use : userNoAccess